xml/docu_styleguide.outline.xml

```diff
@@ -307,6 +296,22 @@
      </para>
     </listitem>
    </varlistentry>
+    <varlistentry>
+     <term>Contributors</term>
+     <listitem>
+      <para>
+       Writing documentation is a collaborative effort. To also give credit to
+       external contributors, add an appendix to the guides, which points to the
+       <guimenu>Contributors</guimenu> page for the respective GitHub
+       repository. For an example, see <xref linkend="sec-contributors"/>.
+      </para>
+      <para>
+       For &suse; Best Practices and other articles and small documents whose
+       content is created and maintained by a single or very few individuals
+       outside the documentation team, credit the authors on the title page.
```

Comment:
```suggestion
       For &suse; Best Practices and other articles and small documents whose
       content is created and maintained by a single author, or very few individuals,
       from outside the documentation team, credit the authors on the title page.
```

Reply:
Just a suggestion because "a single or very few individuals" reads very strangely to me.

Reply:
I agree it reads _a little_ strange but not quite as strange as the suggestion does. The suggestion arbitrarily introduces a distinction between "a single _author_" and "very few _individuals_" as if those roles were somehow different.

Maybe this suggestion, replacing the word "individuals" and trying to make it simpler overall, is better then?

```suggestion
       For &suse; Best Practices and other articles and small documents whose
       content is created and maintained by five or fewer authors, all of which
       from outside the documentation team, credit the authors on the title page.
```

Reply:
OK, good point, and that's fine. I would suggest "all of whom _are_ from outside..." though.